tests/Credentials/ServiceAccountCredentialsTest.php

```diff
@@ -26,30 +26,31 @@
 use Google\Auth\CredentialsLoader;
 use Google\Auth\OAuth2;
 use GuzzleHttp\Psr7;
+use GuzzleHttp\Psr7\Response;
```

Comment:
all imports here need to be sanitized. Some of them are not used at all hence leading to static check failures.